ext/configuration.h

```diff
@@ -247,7 +247,9 @@ enum ddtrace_sampling_rules_format {
     CONFIG(SET, DD_DYNAMIC_INSTRUMENTATION_REDACTED_TYPES, "", .ini_change = zai_config_system_ini_change)     \
     CONFIG(INT, DD_TRACE_BAGGAGE_MAX_ITEMS, "64")                                                              \
     CONFIG(INT, DD_TRACE_BAGGAGE_MAX_BYTES, "8192")                                                            \
-    CONFIG(BOOL, DD_TRACE_INFERRED_PROXY_SERVICES_ENABLED, "false")                                             \
+    CONFIG(BOOL, DD_TRACE_INFERRED_PROXY_SERVICES_ENABLED, "false")                                            \
+    CONFIG(SET, DD_TRACE_HTTP_CLIENT_ERROR_STATUSES, "", .ini_change = zai_config_system_ini_change)           \
+    CONFIG(SET, DD_TRACE_HTTP_SERVER_ERROR_STATUSES, "", .ini_change = zai_config_system_ini_change)           \
```

Comment:
```suggestion
    CONFIG(SET, DD_TRACE_HTTP_CLIENT_ERROR_STATUSES, "500-599", .ini_change = zai_config_system_ini_change)           \
    CONFIG(SET, DD_TRACE_HTTP_SERVER_ERROR_STATUSES, "500-599", .ini_change = zai_config_system_ini_change)           \
```
Then we don't need a fallback behaviour at all.